bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/energy/Home.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api.dto.energy;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * @author Bernhard Kreuz - Initial contribution
+ */
+public class Home implements Serializable {
+
+    private String id;
+    private List<Module> modules = new ArrayList<>();
+    private List<Room> rooms = new ArrayList<>();
```

Comment:
Viewing your usage below, I think you could use a `NAObjectMap` insteal of `List<Module>`